Nurse Tasks/Preview Content/jsonAPI.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  jsonAPI.swift
+//  Nurse Tasks
+//
+//  Created by Wu Maggie on 2023-11-16.
+//
+
+import Foundation
+public struct Task:Codable{
+    var clientName: String
+    var assignedTo : String
+    var location: Address
+    var startTime, endTime: Date
+    var taskTitle, notes, type: String //type for either personal or work task
+    var status: String  //whether the task is completed
+    var reminderEnabled: Bool
+    
+    static let allTasks:[Task] = Bundle.main.decode(file: "data.json")
+    //static let allLocations:[Address] = Bundle.main.decode(file: <#T##String#>)
+    static let sampleTask : Task = allTasks[0]
+}
+
+public struct Address:Codable{
+    var street, city: String
+}
+
+public struct Date:Codable{
+    var year, month, day, hour, mintue: String
```

Comment:
Looks like we still have the typo on 'minute'. I'll merge in for now, but we should fix this

Reply:
Hi Ryan, I've already changed the branch to develop, but for the date structure, what changes you guys wants to make

Reply:
Hi @maggiejieyao I think just change 'mintue' to 'minute', other than that I think we're good if you think we have everything we need to set reminders. I'm actually surprised you didn't get a compile error from calling the struct 'Date'  since there is a built in type called 'Date' in Swift already. Maybe we should rename it from 'Date' to 'CustomDate'. I found this article about working with Dates for reference: https://www.hackingwithswift.com/books/ios-swiftui/working-with-dates#:~:text=As%20you've%20seen%2C%20Swift,second%2C%20timezone%2C%20and%20more.